### DISCUSSION TITLE
Is there a way to access Github community  with in Github.com  I dont seem to see any buttons to navigate to github community

### DISCUSSION THREAD

**Discussion by GaneshTamang** [community/community#112985] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way to access Github community  with in Github.com  I dont seem to see any buttons that will   navigate to github community page i can only  get it by  putting link "https://github.com/community/" on  chrome.
 If there is any ways please guide

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @GaneshTamang! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, a shortcut to our community is now in the right-side fly out navigation for all users:
![image](https://github.com/community/community/assets/14021469/94a1be3b-fc40-4b32-aaf6-d32ed846c129)

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by GaneshTamang** [reply]
thankyou !!! so much finally found it
